### PR TITLE
Improve Gallery component documentation

### PR DESCRIPTION
**Pull request by kefranabg**
See internal discussion [here](https://huggingface.slack.com/archives/C1RCG46LS/p1732589119209059)

Explicitly mention that the Gallery component supports video format

### CODE REVIEW

**Comment by kefranabg** [comment]
> The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/hub/pr_1507). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.

Direct link to the updated page: https://moon-ci-docs.huggingface.co/docs/hub/pr_1507/en/model-cards-components

**Review by julien-c — APPROVED** [review]
ok for me

**Review by Weronika80 — COMMENTED** [review]
Weronika80

File: docs/hub/model-cards-components.md [hunk 1]
```diff
@@ -4,26 +4,11 @@
...
-
-
-<div class="flex justify-center">
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gallery.png"/>
-<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gallery-dark.png"/>
-</div>
+### How to use it?
 
-The `<Gallery/>` component will use your Model Card [widget metadata](/docs/hub/models-widgets-examples#text-to-image) to display the images with each associated prompt. 
+1. Update your Model Card [widget metadata](/docs/hub/models-widgets-examples#text-to-image) to add the medias you want to showcase.
```
Comment (pcuenca) [inline docs/hub/model-cards-components.md:11]:
```suggestion
1. Update your Model Card [widget metadata](/docs/hub/models-widgets-examples#text-to-image) to add the media you want to showcase. 
```

**Comment by julien-c** [comment]
this still planned to be merged @kefranabg?

**Comment by kefranabg** [comment]
> this still planned to be merged @kefranabg?

@julien-c yes, but I don't have the repos rights to merge 😅

**Pull request merged**